README.md

```diff
@@ -61,3 +61,43 @@ docker build -t astral:latest .
 ```
 docker run -p 3000:3000 astral:latest
 ```
+# Logging into vault with OIDC
+
+The rails test's configure the OIDC provider, so if the tests pass,
+you can invoke the oidc login as follows:
```

Comment:
we could also configure the OIDC provider in a rake task which is invoked in devcontainer.json ("postCreateCommand")? That way it's always configured.

Reply:
done

Reply:
the doc needs updating as the tests no longer configure it

Reply:
The tests  no longer configure the provider, but they do still configure the initial user, here:
https://github.com/G-Research/astral/blob/2f35777573ba7e104e4eabde5ee8951678575b61/test/lib/clients/oidc_test.rb#L10-L13

So the doc is still correct.